website/content/en/preview/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -79,10 +79,21 @@ The following cluster configuration will:
 
 {{% script file="./content/en/{VERSION}/getting-started/getting-started-with-karpenter/scripts/step06-add-spot-role.sh" language="bash"%}}
 
+### 4. Install Karpenter
+
 {{% script file="./content/en/{VERSION}/getting-started/getting-started-with-karpenter/scripts/step08-apply-helm-chart.sh" language="bash"%}}
 
+{{% alert title="Warning" color="warning" %}}
+Karpenter creates a mapping between CloudProvider machines and CustomResources in the cluster for capacity tracking. To ensure this mapping is consistent, Karpenter utilizes the following tag keys:
+
+* `karpenter.sh/managed-by`
+* `karpenter.sh/provisioner-name`
+* `kubernetes.io/cluster/${CLUSTER_NAME}`
+
+Because Karpenter takes this dependency, any user that has the ability to Create/Delete these tags will have the ability to orchestrate Karpenter to Create/Delete CloudProvider machines as a side-effect. We recommend that you [configure tag-based IAM policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_tags.html) for any users in your account that might have [CreateTags](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateTags.html)/[DeleteTags](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DeleteTags.html) permissions but should not have [RunInstances](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_RunInstances.html)/[TerminateInstances](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_TerminateInstances.html) permissions.
```

Comment:
Techincally it should be for Creating/Deleting tags on instances, right? 
```suggestion
Because Karpenter takes this dependency, any user that has the ability to Create/Delete these tags on instances will have the ability to orchestrate Karpenter to Create/Delete CloudProvider machines as a side-effect. We recommend that you [configure tag-based IAM policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_tags.html) for any users in your account that might have [CreateTags](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateTags.html)/[DeleteTags](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DeleteTags.html) permissions but should not have [RunInstances](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_RunInstances.html)/[TerminateInstances](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_TerminateInstances.html) permissions.
```

Being as tightly scoped as possible, users only need to restrict the CreateTags/DeleteTags Permissions for ResourceIDs `i-*`

Reply:
> Being as tightly scoped as possible, users only need to restrict the

Isn't that all resource ids? I'm not sure I get your point here

Reply:
You can do create tags for ami-123 and create tags for i-123 but the only one that matters is i-123

Reply:
The same applies for Launch Templates too

Reply:
Still going to ref "CloudProvider machines" to use machines as the lingo for the capacity representation. Updated to mention scoping on the resource type